boards/nordic/nrf54h20dk/nrf54h20dk_nrf54h20_cpuapp.dts

```diff
@@ -343,12 +343,19 @@ zephyr_udc0: &usbhs {
 
 /* Trim this RAM block for making room on all run-time common S2RAM cpu context. */
 &cpuapp_ram0 {
-	reg = <0x22000000 (DT_SIZE_K(32)-32)>;
-	ranges = <0x0 0x22000000 (0x8000-0x20)>;
+	reg = <0x22000000 (DT_SIZE_K(32)-36)>;
```

Comment:
Note that in upstream, there has been a reformatting commit that added spaces around `-`..

Reply:
done